packages/validators-util/src/isValidRegistreringsnummer/isValidRegistreringsnummer.ts

```diff
@@ -1,3 +1,13 @@
-export const REGNR_REGEX = /^[a-zA-Z]{2}\d{5}/;
+export const BIL_REGNR_REGEX = /^[a-zA-Z]{2}\s?\d{5}$/;
+export const MC_MOPED_REGNR_REGEX = /^[a-zA-Z]{2}\s?\d{4}$/;
 
-export const isValidRegistreringsnummer = (value: string): boolean => REGNR_REGEX.test(value);
+interface Props {
+    value: string;
+    product?: "bil" | "mc" | "moped";
+}
+export const isValidRegistreringsnummer = ({ value, product = "bil" }: Props): boolean => {
+    if (!product || product === "bil") {
+        return BIL_REGNR_REGEX.test(value);
+    }
+    return MC_MOPED_REGNR_REGEX.test(value);
+};
```

Comment:
Dette blir den eneste funksjonen hvor parameteret er et objekt. Foreslår at validatorene som standard har verdien som skal valideres som et første parameter, og så kan options følge som parameter(e) etter det:

```suggestion
export interface IsValidRegistreringsnummerOptions {
    vehicle?: "bil" | "mc" | "moped";
}
export const isValidRegistreringsnummer = (
    value: string,
    options: IsValidRegistreringsnummerOptions = { vehicle: "bil" },
): boolean => {
    if (options.vehicle === "bil") {
        return BIL_REGNR_REGEX.test(value);
    }
    return MC_MOPED_REGNR_REGEX.test(value);
};
```

Reply:
Helt enig, mye bedre 👍